### DISCUSSION TITLE
google/gemma-2-27b-it QLORA

### DISCUSSION THREAD

**Discussion by CyberNativeAI** [axolotl-ai-cloud/axolotl#1743] [Q&A]
Hi everyone,

I have tried QLORA fine-tuning gemma-2-27b with chatml and flash-attention yesterday and the result model seems confused, even tho the loss seemed to go down and overall everything looked smooth. May someone please share tips&tricks for fine-tuning this model?

**Comment by NanoCode012** [comment]
Hey, sorry for late response, since it's a qlora, the impact on the weights are not as large as lora / full fine-tuning. Have you tried those other options?

**Reply by NanoCode012** [reply]
Hey, I'll close this discussion for now but if you're still having this issue, please re-open / create a new one.

**Reply by CyberNativeAI** [reply]
It worked out eventually, had to play with parameters. Please close.

**Reply by NanoCode012** [reply]
Thanks for updating us on this!
